src/sanitizer/index.js

```diff
@@ -0,0 +1,15 @@
+import TurndownService from 'turndown';
+import cheerio from 'cheerio';
```

Comment:
>  “[Cheerio](https://www.npmjs.com/package/cheerio) removes all the DOM inconsistencies and browser cruft from the jQuery library, revealing its **truly gorgeous API**.”

😰 

It also says:

> Cheerio wraps around parse5 parser.

Considering the current use we have of `cheerio` is apparently only parsing (as opposed to DOM manipulation), could we consider using directly [`parse5`](https://github.com/inikulin/parse5) instead? 🙏